### PR TITLE
chore: update pr template to align with newsletter

### PR DESCRIPTION
**Pull request by nickofthyme**
## Summary

Update pr template to align with the newsletter

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,12 +1,33 @@
 ## Summary
 
-Summarize your PR.
+<!-- Summarize your PR. This will be included in our newsletter -->
```
Comment (markov00) [inline .github/pull_request_template.md:3]: We can improve the description here a bit by providing some tips and examples like:
```
- The summary is intended for a consumer audience, avoid any internal or implementation details. You can include those in the details section. 
- Generally only `fix:` and `feat:` PRs will be included in the newsletter. Also, PRs with BREAKING CHANGES are added.
- Describe the feature or fix as you would if you were advertising it in the newsletter:
    - ❌ : This commit close the request `#123` and adds the prop `helloWorld` to `Settings`
    - ✅ : The `helloWorld` prop is now available in the `Settings` component to bring joy when rendering the chart.
    - ❌ : Fixing the tooltip position outside the chart area avoiding overflows.
    - ✅ : The tooltip no longer overflows the chart DOM container when using the `tooltip.boundary = 'chart'` in the `Settings` component.
- Add a clear screenshot or animated gif as an example if the change can be understood better and easier with a visual aid.
- If the PR involves a bigger feature, please add more context to it, describing why the feature was added, what actually improve, and how the users can leverage it to improve their data visualizations
- If the PR involves a breaking change include the following part and clearly state which contract is broken:

### BREAKING CHANGE 
The `tooltip.boundary` prop in the `Settings` component now only accepts a single DOM element ID.

```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+
+<!-- screenshot/gif/mpeg-4 for visual changes -->
+
+
+## Details
+
+<!-- Details beyond the summary to explain nuances -->
+
+
+## Connected issues
```
Comment (markov00) [inline .github/pull_request_template.md:15]: I wasn't sure about this title. Probably something like: `Closing issues` `references` can be used instead. Worth having a chat together about it

File: .github/pull_request_template.md (hunk 1 above)
Comment (nickofthyme) [reply .github/pull_request_template.md]: 5b2b698

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+  fix #1108
+-->
+
 
-If it involves visual changes include a screenshot or gif.
 
 ### Checklist
 
-Delete any items that are not applicable to this PR.
+<!-- Delete any items that are not applicable to this PR. -->
```
[the remark this answers is not in the document]
Comment (nickofthyme) [reply .github/pull_request_template.md:46]: 5b2b698

File: .github/pull_request_template.md (hunk 2 above)
Comment (nickofthyme) [reply .github/pull_request_template.md]: 5b2b698

**Review by rshen91 — APPROVED** [review]
I don't have any suggestions on wording as it stands - I think this is very clear and will hopefully make synthesizing the newsletter easier

**Pull request merged**